docs/i18n-l10n/contributing-translations.md

```diff
@@ -112,6 +112,19 @@ When you save a translation, then it is committed on a branch used only for tran
 Maintainers will periodically review the pull request that Weblate creates automatically, and merge it.
 
 
+### Wewblate workflow in Volto
+
+Volto does some checks in all pull-requests that require a changelog entry to be present and the po files to have a given formatting.
+
+Weblate doesn't create such changelog entry and its po style guide does not match the one of Volto, so to merge the changes coming to weblate, a maintainer has to do the following:
```

Comment:
When you say "maintainer", do you mean "Volto Team" member? Otherwise, I'm afraid most people will assume that is only @sneridagh, and it would be good to spread the workload.

```suggestion
Weblate doesn't create a change log entry, and its `.po` file style guide does not match Volto's.
This means that a maintainer has to perform the following tasks to merge the changes coming from Weblate.
```